css/style.css

```diff
@@ -506,3 +524,241 @@ textarea:focus:invalid {
   text-align: center;
   color: rgb(235, 235, 255);
 }
+
+@media screen and (max-width: 992px) {
+  .bootom-shine-1 {
+    position: absolute;
+    width: 150px;
+    height: 150px;
+    left: 600px;
+    filter: blur(0.5px);
+  }
+
+  .bootom-shine-1 > img {
+    position: relative;
+    width: 150px;
+    height: 150px;
+  }
+
+  .chat-container {
+    display: none;
+  }
+
+  .desk-footer {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  .user-name {
+    margin-left: 156px;
+  }
+
+  .right-nav {
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 156px;
+    width: 50%;
+  }
+
+  .right-nav > .link-list-desk {
+    display: flex;
+    justify-content: space-between;
+    padding-left: 0;
+    list-style: none;
+    width: 50%;
+    height: 100%;
+    align-items: center;
+  }
+
+  li > .menus {
+    text-decoration: none;
+    color: white;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .link-humburgur {
+    display: none;
+  }
+
+  .top-shine-1 > img {
+    position: absolute;
+    width: 200px;
+    height: 200px;
+    left: 0.5em;
+  }
+
+  .bootom-shine-1 {
+    position: absolute;
+    width: 200px;
+    height: 200px;
+    top: 435px;
+    left: 1100px;
+    filter: blur(0.5px);
+  }
+
+  .bootom-shine-1 > img {
+    position: relative;
+    width: 200px;
+    height: 200px;
+  }
+
+  .experience {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .skills > .skill-list {
+    display: flex;
+    flex-direction: column;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 13.1389px;
+    line-height: 25px;
+    color: rgb(255, 255, 255);
+    list-style: none;
+  }
+
+  .social-links {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    justify-content: space-around;
+  }
+
+  .social-links > .social-icon {
+    display: flex;
+    justify-content: space-between;
+    height: 50px;
+    min-width: 20%;
+    align-items: center;
+    padding-left: 0;
+  }
+
+  .details > * {
+    display: none;
+  }
+
+  .work-opening {
+    display: flex;
+    flex-direction: column;
+    margin-left: 100px;
+    margin-right: 100px;
+  }
+
+  .works {
+    display: flex;
+    justify-content: space-between;
+    margin-left: 100px;
+    margin-right: 100px;
+  }
+
+  .project-img {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0;
+  }
```

Comment:
- [ ] [REQUIRED] Kindly fix the padding of your `See Project button` so that it has equal padding top, buttom, right, and left. 
- [ ] [REQUIRED] The margin between your cards is not as much as the margin provided in the figma template, please refer to the design to get the margin.
Kindly stick to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) as much as possible.

#### Please click each image to have a better view of their differences.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![3](https://user-images.githubusercontent.com/58692590/150564987-da7b69d3-6eff-410c-aeef-f52f17b844f6.png) | ![33](https://user-images.githubusercontent.com/58692590/150565021-555046a8-04b9-4631-9d8d-7ecc4438a73f.png)